gateway/1.4/modules/ROOT/pages/flex-architecture-uc-overview.adoc

```diff
@@ -0,0 +1,103 @@
+= Integration Use Cases
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+The following integration use cases are determined by the ownership of the API Provider and API Consumer. Each use case shows physical patterns that demonstrate the necessary services in the system. 
```

Comment:
Just checking whether "Provider" and "Consumer" should be lower-case instead of upper-case.